hack/generate-manifest-flow-visibility.sh

```diff
@@ -55,6 +66,22 @@ case $key in
     KEEP=true
     shift
     ;;
+    --volume)
+    VOLUME="$2"
+    shift 2
+    ;;
+    -sc|--storageclass)
+    STORAGECLASS="$2"
+    shift 2
+    ;;
+    --local)
+    LOCALPATH="$2"
+    shift 2
+    ;;
+    --nfs)
+    NFSPATH="$2"
+    shift 2
+    ;;
```

Comment:
I update the shell with the first way. I think it might be easier to do something similar to the second way when we move to Helm, with which we can load this value from `value.yaml`. Will target Helm chart support in release 1.7.